src/general/HexLayout.cs

```diff
@@ -171,6 +171,17 @@ public bool Remove(T hex)
         return true;
     }
 
+    /// <summary>
+    ///   Moves the item from <see cref="index1"/> to <see cref="index2"/> and the item from <see cref="index2"/> to
+    ///   <see cref="index1"/>.
+    /// </summary>
+    /// <param name="index1">The index of one of the items that should be swapped.</param>
+    /// <param name="index2">The index of the other item that should be swapped.</param>
+    public void SwapIndexes(int index1, int index2)
```

Comment:
```suggestion
    public void SwapIndices(int index1, int index2)
```

I think the plural goes like that for index.

Reply:
Yeah, looking it up, indices seems to be the preference outside of North America.